proto/attestation/expected_value.proto

```diff
@@ -0,0 +1,42 @@
+
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.attestation.v1;
+
+import "proto/digest.proto";
+import "proto/attestation/reference_value.proto";
+
+option go_package = "proto/oak/attestation/v1";
+option java_multiple_files = true;
+option java_package = "com.google.oak.attestation.v1";
+
+// A collection of HexDigests that can be
+message HexDigests {
+  repeated HexDigest digests = 1;
+}
+
+message ExpectedDigests {
+  oneof type {
+    // We propagate SkipVerification from ReferenceValues here.
+    SkipVerification skip = 1;
```

Comment:
I would prefer keeping skip verification rather than attaching special meaning to an empty list of digests. If the list of "good" digests are empty I would logically expect none to match and therefore verification to fail rather than pass.

Reply:
Can we just have a separate empty message in expected_value.proto then? This would remove the inter-proto dependency.

Reply:
It is not about the inter-proto dependency per se, that is fine. I feel that introducing the present dependency is even confusing - I'd ask myself, why do expected values depend on reference values, and when tracking it down there is no overlap, since the empty message is "nothing".

Reply:
Agree. My suggestion was just not to reuse the same skip message, but have a separate one.

Reply:
I was on the fence about sharing vs. reusing the SkipVerification message. Sharing them does make sense to me, since there's a strong relationship between reference_value and expected_value family of protos. 

That said, when it comes to proto semantics, making things distinct often ends up being the right choice, so I can be easily convinced to remove the dependency and create a Skip variant just for this file.

The other reason I re-used this one was that these are in the same package, so I'll need to come up with a different name for the proto in this context, otherwise there will be a name collision.